Rest/client.py

```diff
@@ -0,0 +1,31 @@
+import requests
+
+class Task_rest():
+
+    def __init__(self, url, parameter_for_query):
+        self.url = url
+        self.parameter_for_query = parameter_for_query
```

Comment:
Client can work with different query? need to move to parameters to method post, get ...